components/raftstore/src/store/entry_storage.rs

```diff
@@ -1402,6 +1402,7 @@ impl<EK: KvEngine, ER: RaftEngine> EntryStorage<EK, ER> {
 
     pub fn clear(&mut self) {
         self.cache = EntryCache::default();
+        self.term_cache = TermCache::default();
```

Comment:
When will it be cleared?

Reply:
1. `destory` peer;
2. And `apply_snapshot` if the peer is already initialized with old data